solutions/cfgym-102951C.mdx

```diff
@@ -74,4 +74,57 @@ int main() {
 
 </CPPSection>
 
+<JavaSection>
+	
+``` java
+import java.io.*;
+
+import java.util.*;
+
+
+public class Main {
+	//CodeSnip{Kattio}
+	public static void main(String[] args) {
+		Kattio io = new Kattio();
+
+		int n = io.nextInt();		
+		int[] a = new int[n+1], b = new int[n+1], c = new int[n+1], lis = new int[n+1];
+		int[] pos = new int[n+2];
```

Comment:
need consistent operator spacing so `new int[n + 1]`

Reply:
Hi, I've committed in the changes that you told me to make. Please let me know if I have missed anything or If I can improve something.

Reply:
If you've made the requested changes, please re-request reviews as mentioned by stalebot.